industrial_taxonomy/pipeline/glass_clusters/utils.py

```diff
@@ -0,0 +1,122 @@
+"""Utils to create a sector corpus"""
+from toolz import pipe
+from functools import partial
+from typing import Dict, List
+import pandas as pd
+
+from industrial_taxonomy.getters.companies_house import get_sector
+from industrial_taxonomy.getters.glass_house import (
+    get_description_tokens,
+    glass_companies_house_lookup,
+)
+
+NE_CODES = {
+    "CARDINAL",
+    "DATE",
+    "GPE",
+    "LOC",
+    "MONEY",
+    "NORP",
+    "ORDINAL",
+    "ORG",
+    "PERSON",
+    "QUANTITY",
+    "TIME",
+}
+
+
+def strip_nes(tokenised_dict: Dict[int, List[str]]) -> Dict[int, List[str]]:
+    """Removes named entities from company tokenised descriptions"""
+
+    return {
+        k: [t for t in v if all(ne not in t for ne in NE_CODES)]
+        for k, v in tokenised_dict.items()
+    }
+
+
+def reverse_dict(_dict: dict) -> Dict:
+    """Turns keys into values and viceversa"""
+    return {v: k for k, v in _dict.items()}
+
+
+def gl_sic4_lookup() -> dict:
+    """Creates a lookup between glass_ids and 4 digit SIC codes"""
+
+    return (
+        get_sector()
+        .assign(
+            glass_id=lambda df: df.index.map(
```

Comment:
I'd suggest `glass_id` -> `org_id` to keep naming conventions consistent (even if `org_id` is probably a worse name)